internal/server/proto/server.proto

```diff
@@ -849,6 +864,13 @@ message Ref {
     string id = 1;
     string name = 2;
   }
+
+  // Trigger references a Trigger message value to be used for a given operation.
+  // It can be looked up by name or id. Id takes priority if both are set.
+  message Trigger {
+    string id = 1;
+    string name = 2;
```

Comment:
We should use a `oneof` here.

Reply:
Now that I've fleshed out the CLI a bit, I'm thinking of dropping `name` completely and just referring to name. Thoughts? Mostly because we probably don't want to make `name` unique across server installations, just `id`.

Reply:
"just referring to ID"* you mean? I think that is better. We have a "description" already for human frindly understanding of why a trigger exists. And ID satisfies our security-through-obscurity for non-authenticated triggers.

Reply:
Oops, typo! Yep, meant just referring to *id*.